docs/builtin-plugins.md

```diff
@@ -4,22 +4,72 @@ Fusebox contains premade plugins, that should help you to get started.
 
 ## CSS Plugin
 
-It's very easy to start working with css files. You have 2 options, you either bundle the contents or serve the files. A decision that can be made at build time.
+The CSS plugin is elegant and powerful.  It can be used to combine all CSS files into a single bundle, or to mark files as external to
+the bundle, to be served separately by the web server.  By default, all CSS files are bundled.
 
 For example:
 ```js
 plugins: [
+    fsbx.CSSPlugin({})
+]
+```
+
+### Chaining CSS plugins to transform SCSS, Sass, and others
+
+Fusebox's plugin system allows simple transformation of more complex CSS systems through plugin chaining.  For example, to transform
+Sass into CSS:
+
+```
+plugins: [
+    [
+        fsbx.SassPlugin({ outputStyle: 'compressed' }),
+        fsbx.CSSPlugin({ write: true })
+    ]
+]
+```
+
+* This simple code will change any scss file to a css file - `./main.scss` becomes `build/main.css` (your [outFile](#out-file) folder + project path)
+* it also creates `main.css.map` if sourcemaps have been specified
+* it automatically appends the new CSS files to the head of the web page and serves them
+
+A more extensive example can be found [here](https://github.com/fuse-box/angular2-example)
+
+### Options
```

Comment:
This is a heading.  All headings are upper-cased.  The things below are actual code options, and are lower-cased in the code.